taskcluster/scripts/toolchain/android-gradle-dependencies.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
nit: Do we add license headers to those files?

Reply:
Oops! Done.